### PR TITLE
fix pelican

### PR DESCRIPTION
**Pull request by ahmadia**
relies on https://github.com/hashdist/hashstack/pull/682

### CODE REVIEW

**Comment by ahmadia** [comment]
Waiting on Travis to double-check that upgrading Jinja doesn't breaking things.

File: pkgs/python-markdown.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+extends: [setuptools_package]
+
+dependencies:
+  build: []
+  run: []
+
+sources:
+ - url: https://pypi.python.org/packages/source/M/Markdown/Markdown-2.6.zip
+   key: zip:twdfrwawe3ad52oz37f3kfryfxmqmtrb
```
Comment (certik) [inline pkgs/python-markdown.yaml]: flip url and key

File: pkgs/typogrify.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+extends: [setuptools_package]
+
+dependencies:
+  build: []
+  run: []
+
+sources:
+ - url: https://pypi.python.org/packages/source/t/typogrify/typogrify-2.0.7.tar.gz
+   key: tar.gz:rpsgndg2inawhtrctwd4ujz2cgjczmlb
```
Comment (certik) [inline pkgs/typogrify.yaml]: flip url and key

**Comment by certik** [comment]
@ahmadia you should just use `hit fetch url` and copy & paste.

**Comment by ahmadia** [comment]
@certik - Those come from `skeleton-pypi`, which has the old order.

**Comment by ahmadia** [comment]
Looks like OpenSSL was down.  My primary concern was Jinja, so I'm merging this.

**Comment by certik** [comment]
@ahmadia then you should flip the url and key, *and* change `skeleton-pypi`. ;)

**Pull request merged**